WordPress/gutenberg
# Avatar: add padding support

Related:

- https://github.com/WordPress/gutenberg/issues/43241
- https://github.com/WordPress/gutenberg/issues/43243

## What?

Add padding support to the Avatar block.

## Why?

To create consistency across blocks.

## How?

Added the relevant block support in `block.json`

## Testing Instructions

1. Create a new post
2. Insert a new Avatar.
3. Adding Padding and configure.

Also, test in Global Styles as well.

1. Go to Appearance > Editor (beta)
2. Add an Avatar block to the template
3. Navigate to the Global Styles and choose Avatar block
4. Configure padding and verify

![avatar-block-padding](https://user-images.githubusercontent.com/405912/186162973-6ae17b47-d47c-4927-b01b-5cac22df370b.gif)

<img width="1714" alt="Screen Shot 2022-08-23 at 8 11 02 AM" src="https://user-images.githubusercontent.com/405912/186155116-465f7ba3-291e-4c8e-bfc6-b30092c3f8b0.png">
